api/types/swarm/secret.go

```diff
@@ -0,0 +1,30 @@
+package swarm
+
+// Secret represents a secret.
+type Secret struct {
+	ID string
+	Meta
+	Spec       *SecretSpec `json:",omitempty"`
+	Digest     string      `json:",omitempty"`
+	SecretSize int64       `json:",omitempty"`
```

Comment:
We probably don't want `omitempty` for `SecretSize`. Probably not for any of these fields, actually.